### PR TITLE
Bump to 21w06a

### PR DESCRIPTION
**Pull request by gabizou**
Just to show the changes from 1.16.5 to Minecraft Snapshot 21w06a.

### CODE REVIEW

File: src/accessors/java/org/spongepowered/common/accessor/server/level/ServerPlayerGameModeAccessor.java [hunk 1]
```diff
@@ -36,4 +37,6 @@
 
     @Accessor("destroyPos") BlockPos accessor$destroyPos();
 
+    @Accessor("level") ServerLevel accessor$getLevel();
```
Comment (Zidane) [inline src/accessors/java/org/spongepowered/common/accessor/server/level/ServerPlayerGameModeAccessor.java]: Violates accessor naming policy, should be accessor$level()

File: src/accessors/java/org/spongepowered/common/accessor/world/level/LevelAccessor.java [hunk 2]
```diff
@@ -24,18 +24,11 @@
...
-import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.asm.mixin.gen.Accessor;
-
-import java.util.List;
 import net.minecraft.world.level.Level;
-import net.minecraft.world.level.block.entity.BlockEntity;
+import org.spongepowered.asm.mixin.Mixin;
 
 @Mixin(Level.class)
 public interface LevelAccessor {
```
Comment (Zidane) [inline src/accessors/java/org/spongepowered/common/accessor/world/level/LevelAccessor.java]: This accessor needs to be deleted if nothing is left in it.

File: src/accessors/java/org/spongepowered/common/accessor/world/level/block/entity/AbstractFurnaceBlockEntityAccessor.java [hunk 3]
```diff
@@ -51,4 +57,9 @@
 
     @Accessor("recipeType") RecipeType<? extends AbstractCookingRecipe> accessor$recipeType();
 
+    @Invoker("canBurn")
+    static boolean invoker$canBurn(@Nullable Recipe<?> var0, NonNullList<ItemStack> var1, int var2) {
+        throw new UnsupportedOperationException("Untransformed shadow");
```
Comment (Zidane) [inline src/accessors/java/org/spongepowered/common/accessor/world/level/block/entity/AbstractFurnaceBlockEntityAccessor.java]: There is an exception to throw for this, UntransformedInvokerError

File: src/accessors/java/org/spongepowered/common/accessor/world/level/entity/EntityTickListAccessor.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.level.entity.EntityTickList;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(EntityTickList.class)
+public interface EntityTickListAccessor {
+
+    @Accessor("active") Int2ObjectMap<Entity> accessor$getActive();
```
Comment (Zidane) [inline src/accessors/java/org/spongepowered/common/accessor/world/level/entity/EntityTickListAccessor.java]: accessor$active()

File: src/accessors/java/org/spongepowered/common/accessor/world/level/entity/PersistentEntitySectionManagerAccessor.java [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+import net.minecraft.world.level.entity.EntityAccess;
+import net.minecraft.world.level.entity.EntitySectionStorage;
+import net.minecraft.world.level.entity.PersistentEntitySectionManager;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(PersistentEntitySectionManager.class)
+public interface PersistentEntitySectionManagerAccessor<T extends EntityAccess> {
+
+    @Accessor("sectionStorage") EntitySectionStorage<T> accessor$getSectionStorage();
```
Comment (Zidane) [inline src/accessors/java/org/spongepowered/common/accessor/world/level/entity/PersistentEntitySectionManagerAccessor.java]: accessor$sectionStorage()

File: src/accessors/java/org/spongepowered/common/accessor/world/level/entity/TransientEntitySectionManagerAccessor.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+import net.minecraft.world.level.entity.EntityAccess;
+import net.minecraft.world.level.entity.EntitySectionStorage;
+import net.minecraft.world.level.entity.TransientEntitySectionManager;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(TransientEntitySectionManager.class)
+public interface TransientEntitySectionManagerAccessor<T extends EntityAccess>{
+
+    @Accessor("sectionStorage") EntitySectionStorage<T> accessor$getSectionStorage();
```
Comment (Zidane) [inline src/accessors/java/org/spongepowered/common/accessor/world/level/entity/TransientEntitySectionManagerAccessor.java]: accessor$sectionStorage()

File: src/accessors/java/org/spongepowered/common/accessor/world/level/levelgen/NoiseGeneratorSettingsAccessor.java [hunk 7]
```diff
@@ -35,8 +35,15 @@
 @Mixin(NoiseGeneratorSettings.class)
 public interface NoiseGeneratorSettingsAccessor {
 
-    @Invoker("<init>") static NoiseGeneratorSettings invoker$new(final StructureSettings structureSettings, final NoiseSettings noiseSettings,
-            final BlockState defaultBlock, final BlockState defaultFluid, final int bedrockRoofY, final int bedrockFloorY, final int seaLevel, final boolean disableMobGeneration) {
+    @Invoker("<init>") static NoiseGeneratorSettings invoker$new(
```
Comment (kashike) [inline src/accessors/java/org/spongepowered/common/accessor/world/level/levelgen/NoiseGeneratorSettingsAccessor.java:38]: Missing `final` for parameters now.

File: src/accessors/java/org/spongepowered/common/accessor/world/level/block/entity/AbstractFurnaceBlockEntityAccessor.java (hunk 3 above)
Comment (gabizou) [reply src/accessors/java/org/spongepowered/common/accessor/world/level/block/entity/AbstractFurnaceBlockEntityAccessor.java]: Blame copy paste from the shadow that existed before

**Pull request closed without merging**